js/src/components/Footnotes.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+import { withStyles } from '@material-ui/core';
+import LessonGrid from '@anu/components/LessonGrid';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+
+const StyledBox = withStyles((theme) => ({
+  root: {
+    marginBottom: theme.spacing(4),
+  },
+}))(Box);
+
+const useStyles = makeStyles((theme) => ({
+  divider: {
+    width: '100%',
```

Comment:
Div does not need width 100% - it's set to the width of the container by default.

Reply:
Ok.